app/views/users/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<section>
+ <% @users.each do |u| %>
```

Comment:
- [ ] Kindly update the number of posts and the number of comments for all views to match the wireframe as now all numbers are zero.
![image](https://github.com/porag-m06/Rails-Blog-App/assets/24703171/7268fef5-b142-4590-92e6-533fe5a5cb6a)
![image](https://github.com/porag-m06/Rails-Blog-App/assets/24703171/21400eee-f6bc-45cd-b35f-5f07b52c4d84)